src/plugins/intel_cpu/src/nodes/transpose.cpp

```diff
@@ -122,6 +148,53 @@ void MKLDNNTransposeNode::prepareParams() {
     params.src_block_dims = srcDesc->getBlockDims();
     auto dstDesc = getChildEdgeAt(0)->getMemory().GetDescWithType<BlockedMemoryDesc>();
     params.dst_block_dims = dstDesc->getBlockDims();
```

Comment:
minor
seems 147 - 150 can be executed after check on `performAsReorder`, so we no need to call it on each `prepareParams()` call